include/af/device.h

```diff
@@ -363,17 +363,28 @@ extern "C" {
                                     size_t *lock_bytes, size_t *lock_buffers);
 
 #if AF_API_VERSION >= 33
-    ///
-    /// Prints buffer details from the ArrayFire Device Manager
-    //
-    /// \param [in] msg A message to print before the table
-    /// \param [in] device_id print the memory info of the specified device.
-    ///  -1 signifies active device.
-    ///
-    /// return AF_SUCCESS if successful
-    ///
-    /// \ingroup device_func_mem
-    ///
+    /**
+       Prints buffer details from the ArrayFire Device Manager.
+
+       The result is a table with several columns:
+
+        * POINTER:   The hex address of the array's device or pinned-memory
+                     pointer (depending on the backend)
```

Comment:
We can remove the "depending on backend" is not backend dependent per say

Reply:
Fixed.